### PR TITLE
chore: merge `dev` to `main`

### PR DESCRIPTION
**Pull request by PalmDevs**
Finally, :trollface:

### CODE REVIEW

**Review by Aunali321 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -135,4 +135,7 @@ app.*.map.json
 !/dev/ci/**/Gemfile.lock
 
 Firebase related
-.firebase
\ No newline at end of file
+.firebase
+
+# Depdenency directories
```
Comment (EvadeMaster) [inline .gitignore:140]: Should be

```
# Dependency directories
```

File: .gitignore [hunk 2]
```diff
@@ -135,4 +135,7 @@ app.*.map.json
 !/dev/ci/**/Gemfile.lock
 
 Firebase related
```
Comment (EvadeMaster) [inline .gitignore:137]: Mark this as comment

```
# Firebase related
```

**Comment by Ushie** [comment]
#810 should be merged first

File: assets/i18n/en_US.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
         "settingsTab": "Settings"
     },
     "homeView": {
+        "refreshSuccess": "Refresh successfully",
```
Comment (EvadeMaster) [inline assets/i18n/en_US.json:16]: Shouldn't this be "Successfully refresh"?

File: assets/i18n/en_US.json [hunk 4]
```diff
@@ -47,15 +48,16 @@
...
+        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported. Proceed anyways?"
     },
     "appSelectorCard": {
         "widgetTitle": "Select an application",
         "widgetTitleSelected": "Selected application",
         "widgetSubtitle": "No application selected",
         "noAppsLabel": "No applications found",
         "currentVersion": "Current",
-        "recommendedVersion": "Recommended",
+        "recommendedVersion": "Suggested",
```
Comment (EvadeMaster) [inline assets/i18n/en_US.json:60]: Everything related to `recommendedVersion` should be `suggestedVersion` instead

File: assets/i18n/en_US.json (hunk 3 above)
Comment (PalmDevs) [reply assets/i18n/en_US.json:16]: Should either be
 - Successfully refreshed
 - Refreshed successfully

Comment (Ushie) [reply assets/i18n/en_US.json:16]: do Refreshed successfully

File: assets/i18n/en_US.json (hunk 4 above)
Comment (PalmDevs) [reply assets/i18n/en_US.json:60]: u fix :trollface:

**Review by EvadeMaster — APPROVED** [review]
Looks good to me! 🥂

**Comment by PalmDevs** [comment]
I love force-pushes.

File: lib/ui/views/patcher/patcher_viewmodel.dart [hunk 5]
```diff
@@ -82,6 +85,43 @@ class PatcherViewModel extends BaseViewModel {
...
+                Navigator.of(context).pop();
+                navigateToInstaller();
+              },
+            )
+          ],
+        ),
+      );
+    } else {
+      // navigate To Installer
+      navigateToInstaller();
```
Comment (EvadeMaster) [inline lib/ui/views/patcher/patcher_viewmodel.dart:121]: huh? :trollface:

File: CHANGELOG.md
Comment (EvadeMaster) [inline CHANGELOG.md:1]: @PalmDevs is this intended?

Comment (PalmDevs) [reply CHANGELOG.md:1]: Not sure, so I pushed a non-changed one. You want it to be purged?

**Review by Ushie — APPROVED** [review]

**Pull request merged**